grunttasks/experiments.js

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  var PACKAGE_JSON = '../package.json';
+
+  /**
+   *
+   */
+  grunt.registerTask('experiments', 'Updates experiment configuration for train deploys', function () {
+    var fs = require('fs');
+    var semverUtils = require('semver-utils');
+    var lib = require('./lib/experiments');
+
+    var version = require(PACKAGE_JSON).version;
```

Comment:
I think we're already injecting package.json in the [Gruntfile.js](https://github.com/mozilla/fxa-content-server/blob/c2f05420f6444a4e5ad3c9a0aca2c0b0c5cdb518/Gruntfile.js#L31-L33):

```js
  grunt.initConfig({
    pkg: grunt.file.readJSON('package.json')
  });
```

Not sure if you can nuke line 6, 12, 16, and then change line 17 below to somethin' like:
```js
var semver = semverUtils.parse('<%= pkg.version %>');
```